buildSrc/src/main/java/org/opensearch/gradle/DistributionDownloadPlugin.java

```diff
@@ -84,6 +84,8 @@
     private NamedDomainObjectContainer<OpenSearchDistribution> distributionsContainer;
     private NamedDomainObjectContainer<DistributionResolution> distributionsResolutionStrategiesContainer;
 
+    private static String customUrl;
```

Comment:
Name consistently, I would just do `distributionUrl` and `setDistributionUrl` since it's downloading ... a distribution.

Reply:
Sure, I will rename them accordingly